addons/im_livechat/static/src/legacy/public_livechat_chatbot.js

```diff
@@ -553,7 +553,7 @@ const QWeb = core.qweb;
      * @private
      */
     _isLastMessageFromCustomer() {
-        const lastMessage = this.messaging.publicLivechatGlobal.messages.length !== 0 ? this.messaging.publicLivechatGlobal.messages[this.messaging.publicLivechatGlobal.messages.length - 1] : null;
+        const lastMessage = this.messaging.publicLivechatGlobal.messages.length !== 0 ? this.messaging.publicLivechatGlobal.lastMessage : null;
```

Comment:
the condition could probably be removed here since it is now handled in the compute

Reply:
changes done